tensorflow/g3doc/tutorials/mnist/beginners/index.md

```diff
@@ -223,7 +223,7 @@ More compactly, we can just write:
 
 $$y = \text{softmax}(Wx + b)$$
 
-Now let's turn that into something that Tensorflow can use.
+Now let's turn that into something that TensorFlow can use.
```

Comment:
I can see some other instances of "Tensorflow" in this file. If you want to fix this, let's do it thoroughly please?